Adding design proposal around Upgrading a Cluster

lizsurette opened the pull request:
This includes selecting the Hosts from that cluster to upgrade, defining some upgrade options, and then reviewing the hosts and VMs that will be affected. It also includes monitoring this progress view toast notifications and the notification drawer.
https://github.com/lizsurette/ovirt-design/blob/f5f7d0edd500a0a056a90c2880207ca3c216ae9d/admin-ui/compute/upgrade-cluster.md

lizsurette committed:
Adding design proposal around Upgrading a Cluster

This includes selecting the Hosts from that cluster to upgrade, defining some upgrade options, and then reviewing the hosts and VMs that will be affected. It also includes monitoring this progress view toast notifications and the notification drawer.

mykaul commented:
I don't understand what the 'name' button (sort by?) is for in the host selection dialog. You can't just click on the title of the column to sort by it?

mykaul commented:
The 'Timeout' option should go under 'Advanced', hidden by default - the user doesn't know anyway which value to put there. So if 1200 is not good, should he try 1400?

mykaul reviewed (commented):
The upgrade eventually take place in the 'background' - the wizard is closed?

lizsurette commented:
@gregsheremeta @jelkosz Take a look through and feel free to add any comments!

lizsurette commented:
@mykaul 
> I don't understand what the 'name' button (sort by?) is for in the host selection dialog. You can't just click on the title of the column to sort by it?

This is just following the [sorting pattern in PatternFly](https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html?selectedKind=patternfly-react%2FForms%20and%20Controls%2FSort&selectedStory=Sort&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs) as it exists today. It would have any of the column header names in the dropdown to sort by.

> The 'Timeout' option should go under 'Advanced', hidden by default - the user doesn't know anyway which value to put there. So if 1200 is not good, should he try 1400?

Totally get your point! Do you think it makes sense to have an advanced collapsable section here even if there is only one item? 

> The upgrade eventually take place in the 'background' - the wizard is closed?

Exactly. The user will select "Upgrade" and the wizard will be closed while a toast notification will be shown letting the user know that this has started. They can open the notification drawer at any time to view it running as a task, but will also be notified via toast when it's successful or if anything fails. I will make this more clear in the write up.

mykaul commented:
- We don't currently use that button for sorting in oVirt - we just click on the relevant header and it is sorted by it. Also looks cleaner to me?

- We'll have more Advanced fields in the future, most likely... I'd put 'use_maintenance_policy' and 'check_upgrade' there as well, for example.

lizsurette committed:
User Portal Redesign - Login Page

This PR shows an initial proposal for the Login Page for the User Portal. This would make use of the new PatternFly Login: https://github.com/patternfly/patternfly-react/pull/362

lizsurette committed:
Revert "User Portal Redesign - Login Page"

This reverts commit 77643a01694865c71ec67876d9014757b00dd562.
